pyensembl/ensembl_release.py

```diff
@@ -120,6 +123,33 @@ def run_query():
             return results
         return cached_object(pickle_path, compute_fn=run_query)
 
+    def download_all(self):
+        self.download_annotations()
+        self.download_transcripts()
+
+    def download_annotations(self):
```

Comment:
Sure, I think that makes sense. Per my comment above, I could imagine multiple levels of granularity for forcing (download vs. DB setup), but I think it'd be simplest to fully delete and start from scratch.